api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/reshaping/RepeatVector.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 JetBrains s.r.o. and Kotlin Deep Learning project contributors. All Rights Reserved.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlinx.dl.api.core.layer.reshaping
+
+import org.jetbrains.kotlinx.dl.api.core.KGraph
+import org.jetbrains.kotlinx.dl.api.core.layer.Layer
+import org.jetbrains.kotlinx.dl.api.core.shape.TensorShape
+import org.tensorflow.Operand
+import org.tensorflow.Shape
+import org.tensorflow.op.Ops
+import org.tensorflow.op.core.Tile
+
+/**
+ * Layer that repeats the input [n] times.
+ *
+ * Input shape: `2D tensor of shape (num_samples, features)`.
+ *
+ * Output shape: `3D tensor of shape (num_samples, n, features)`.
+ *
+ * @property n Repetition factor.
+ * @property [name] Custom layer name.
+ * @constructor Creates [RepeatVector] object.
+ *
+ * @author Stan van der Bend
+ * @since 0.2
+ */
+public class RepeatVector(
+    public val n: Int,
+    name: String = ""
+) : Layer(name) {
+
+    override fun build(tf: Ops, kGraph: KGraph, inputShape: Shape) {
```

Comment:
I propose
```kotlin
override fun build(tf: Ops, kGraph: KGraph, inputShape: Shape) = Unit
```
as this approach describes itself and doesn't need any comments that brings nothing to reader (as Kotlin gives us the bility to write such code, not like in Java)

Reply:
@avan1235 Nice! I never thought of that. I copied that piece from another similar Layer class. We might wanna convert all of the cases to this. It's a lot cleaner.